config.ini.sample

```diff
@@ -2,6 +2,7 @@
 apikey=xxxxxxxxx-xxxxx-xxxxx-xxxxxxxxxxx # pragma: allowlist secret
 service_url=https://....
 use_bearer_token=False
+;max_threads=20
```

Comment:
Please add space above/below to break up the logical groups.
This would have been helpful before but even more important with three logical groups (connection, concurrency, model configuration)

Reply:
Done. Let me know if I misinterpreted the request